src/jubilation-line/index.jsx

```diff
@@ -0,0 +1,31 @@
+// @flow
+import React from 'react';
+import getContext from '../helpers/context';
+import Animation from '../jubilation-animation';
+
+type Props = { x1?: number, x2?: number, y1?: number, y2?: number};
```

Comment:
I think we need to make all of these required props. No way to draw a line without them
ie `type Props = { x1: number, x2: number, y1: number, y2: number};`